src/universal/utils/fromNow.js

```diff
@@ -26,3 +26,16 @@ export default function fromNow(time) {
   // this is both for eslint, and for chuckles. It should never happen:
   return 'infinitely long ago';
 }
+
+export function refreshPeriod(time) {
+  const msElapsed = (Date.now() - time) || 0;
+  const threshKeys = Object.keys(thresholds);
+  for (let i = 1; i < threshKeys.length; i++) {
+    const currentThresh = thresholds[threshKeys[i]];
+    if (msElapsed < currentThresh) {
+      return i === 1 ? 30 * thresholds.second : thresholds[threshKeys[i - 1]];
```

Comment:
Challenge:
I have a project that is 1 hr 10 mins old.
The fromNowString will be `1 hour ago`.
With the current logic, I will rerender every minute, which is perfectly reasonable, but it results in 49 pointless rerenders.
Is it possible to return a refreshPeriod of 50 minutes so I only rerender when necessary?